### PR TITLE
Updated to VERA++ 1.3.0 in Travis and changed files to comply with style checks

### PR DESCRIPTION
**Pull request by hakonsbm**
In PR #691 all files were made compliant with style checks. However, because of a bug in VERA++ 1.2.1, a `#pragma omp for` directive is seen as a standard for loop. This was a problem in the file `connection_manager.cpp`, and VERA++ was therefore kept ignored in the static code analysis. Because of this, some new style violations were introduced by other PRs.

This PR
  - corrects the style violations
  - In Travis, switches from installing a packaged VERA++ 1.2.1 to installing VERA++ 1.3.0 from source (as this version isn't available as a package)
  - makes the static code analysis not ignore VERA++

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -101,9 +101,15 @@ if [ "$xSTATIC_ANALYSIS" = "1" ] ; then
   echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
 
   echo "MSGBLD0010: Initializing VERA++ static code analysis."
+  git clone https://github.com/verateam/vera.git
```
Comment (tammoippen) [inline build.sh]: Vera++ is actually hosted at bitbucket: http://bitbucket.org/verateam/vera . The GitHub version is a clone. Further, cloning from that repo will give you whatever version is at branch master latest. This is not version 1.3.0 (which was [released 2015-01-22](https://bitbucket.org/verateam/vera/wiki/Home)). The last commit on master is also [not stable](https://bitbucket.org/verateam/vera/commits/cf13c413f641b5816a4fc9cd4b2fb248c81bf2bd). You should use the sources from the 1.3.0 download page: https://bitbucket.org/verateam/vera/wiki/Installation

**Comment by hakonsbm** [comment]
@tammoippen I have made the changes you requested.

**Comment by janhahne** [comment]
@hakonsbm The merged PR #858 changed some of the code of `rate_neuron_ipn_impl.h` and (unfortunately) also introduced some more missing {} blocks in `rate_transformer_node.h` and `rate_transformer_node_impl.h`. I was not aware of the problem, otherwise I would have added the blocks myself in #858, sorry about that!

**Comment by hakonsbm** [comment]
@janhahne No problem, I'll take care of it.

**Pull request merged**